src/status_im/ui/screens/wallet/assets/views.cljs

```diff
@@ -14,42 +11,49 @@
             [status-im.ui.screens.wallet.assets.styles :as assets.styles]
             [status-im.ui.screens.wallet.main.styles :as main.styles]
             [status-im.ui.components.tabs.views :as tabs]
-            [status-im.ui.components.list.views :as list]
-            [status-im.ui.components.styles :as components.styles]))
+            [status-im.ui.components.styles :as components.styles]
+            [status-im.ui.screens.wallet.transactions.views :as transactions]
+            [status-im.utils.utils :as utils]
+            [status-im.ui.components.chat-icon.styles :as chat-icon.styles]
+            [status-im.utils.money :as money]))
 
 
 (defview my-token-tab-title [active?]
-  (letsubs [ {:keys [token-symbol]} [:token-balance]]
+  (letsubs [ {:keys [symbol]} [:token-balance]]
     [react/text {:uppercase? true
                  :style      (assets.styles/tab-title active?)}
-     (i18n/label :t/wallet-my-token {:symbol token-symbol})]))
+     (i18n/label :t/wallet-my-token {:symbol symbol})]))
 
 (defview my-token-tab-content []
   (letsubs [syncing? [:syncing?]
-            {:keys [token-symbol
-                    token-value
+            {:keys [symbol
+                    amount
+                    decimals
                     usd-value]} [:token-balance]]
     [react/view components.styles/flex
      [react/view {:style assets.styles/total-balance-container}
       [react/view {:style assets.styles/total-balance}
-       [react/text {:style assets.styles/total-balance-value} token-value]
-       [react/text {:style assets.styles/total-balance-currency} token-symbol]]
+       [react/text {:style assets.styles/total-balance-value} (money/to-fixed (money/token->unit (or amount 0) decimals))]
```

Comment:
Not 100% sure but I think token->unit deals with amount being nil? So might not need the (or amount 0)